arelle/plugin/validate/ESEF/resources/authority-validations.json

```diff
@@ -0,0 +1,278 @@
+    },
+    "FR": {
+        "name": "France",
+        "reportPackageMaxMB": "40",
+        "reportPackageMeasurement": "zipped"
+    },
+    "HR": {
+        "name": "Croatia"
+    },
+    "HU": {
+        "name": "Hungary"
+    },
+    "IE": {
+        "name": "Ireland"
+    },
+    "IS": {
+        "name": "Iceland"
+    },
+    "IT": {
+        "name": "Italy"
+    },
+    "LI": {
+        "name": "Liechtenstein"
+    },
+    "LT": {
+        "name": "Lithuania"
+    },
+    "LU": {
+        "name": "Luxembourg"
+    },
+    "LV": {
+        "name": "Latvia"
+    },
+    "MT": {
+        "name": "Malta",
+        "reportPackageMaxMB": "25",
+        "reportPackageMeasurement": "zipped"
+    },
+    "NL": {
+        "name": "Netherlands",
+        "reportPackageMaxMB": "100",
+        "reportPackageMeasurement": "unzipped",
+        "extensionAbstractContexts": "allowed",
+        "reportFileNamePattern": "{base}-{date}.xhtml",
+        "reportFileNameRegex": "(.{1,})-[0-9]{4}-[0-9]{2}-[0-9]{2}[.]xhtml$",
+        "extensionElementsAnchoring": "include abstract",
+        "dimension-default-arc-ELRs": "990000 only",
+        "LC3AllowCapitalsInWord": false
+    },
+    "NO": {
+        "name": "Norway"
+    },
+    "PL": {
+        "name": "Poland"
+    },
+    "PT": {
+        "name": "Portugal"
+    },
+    "RO": {
+        "name": "Romania"
+    },
+    "SE": {
+        "name": "Sweden",
+        "reportPackageMaxMB": "100",
+        "reportPackageMeasurement": "zipped"
+    },
+    "SI": {
+        "name": "Slovenia",
+        "reportPackageMaxMB": null
+    },
+    "SK": {
+        "name": "Slovakia"
+    },
+    "UK": {
+        "name": "United Kingdom",
+        "authorityName": "FRC",
+        "comment": "Pertains to UKSEF Guidance",
+        "reference": "https://www.frc.org.uk/getattachment/d41a2831-6626-40b5-86c0-97627b50dd10/FRC-UKSEF-Tagging-Guide-2021-Version-1-1-January-2022.pdf",
+        "reportPackageMaxMB": "150",
+        "reportPackageMeasurement": "zipped",
+        "identiferScheme": "http://www.companieshouse.gov.uk/",
+        "ixTargetUsage": "error",
+        "extensionAbstractContexts": "allowed",
+        "G3.4.2": "ineffective",
+        "G3.4.6": "ineffective",
+        "outdatedTaxonomyURLs": [
+        ],
+        "effectiveTaxonomyURLs": [
+            "http://www.esma.europa.eu/taxonomy/2019-03-27/esef_cor.xsd",
+            "https://www.esma.europa.eu/taxonomy/2019-03-27/esef_cor.xsd",
+            "http://www.esma.europa.eu/taxonomy/2020-03-16/esef_cor.xsd",
+            "https://www.esma.europa.eu/taxonomy/2020-03-16/esef_cor.xsd"
+        ],
+        "authorityRequiredTaxonomyURLs": [
+            "https://xbrl.frc.org.uk/uksef/2021-01-01/uksef-2021-01-01_cor.xsd"
+        ],
+        "additionalMandatoryTags": [
+            "uksef:UKCompaniesHouseRegisteredNumber",
+            "uksef:BalanceSheetDate",
+            "bus:UKCompaniesHouseRegisteredNumber",
+            "bus:BalanceSheetDate",
```

Comment:
These concepts, specifically `UKCompaniesHouseRegisteredNumber` and `BalanceSheetDate`, seem to be duplicates. Is there a UKSEF version of these?